src/robot/htmldata/libdoc/libdoc.html

```diff
@@ -322,7 +324,7 @@ <h2 id="Tags">Tags</h2>
            title="Show tests with this tag">${$value}</a> &middot;
         {{/each}}
         <a href="javascript:resetKeywords()" class="normal-first-letter"
-           title="Show all tests">[Reset]</a>
+           title="Show all tags">[Reset]</a>
```

Comment:
:+1:

Reply:
context where these are selected has head of "Tags" so, reset's title makes more sense than "keywords" - imho